src/backend/expungeservice/request/request.py

```diff
@@ -0,0 +1,26 @@
+from flask import g
+import os
+from expungeservice.database import Database
+
+
+def before():
+
+    host = os.environ['PGHOST']
+    port = os.environ['PGPORT']
+    name = os.environ['PGDATABASE']
+    username = os.environ['PGUSER']
+    password = os.environ['POSTGRES_PASSWORD']
+
+    g.database = Database(
+        host=host, 
+        port=port, 
+        name=name, 
+        username=username, 
+        password=password)
+
+def teardown(exception):
+    try: 
+        g.database.close_connection()
+    except: pass
```

Comment:
I don't think this is a good idea as it will pass on all exceptions. Do you think we can get this under test, and catch the specific exception's?

Reply:
If an exception does occur while trying to close the connection perhaps we can log it? I'm not familiar with how to do that in a python/flask environment, are you?

Reply:
I can redo this to just check there is an open connection to be closed, without a try-except block. The only failure conditions that occurred to me are that the connection didn't get opened for some reason which is easy to check for. My thinking here was that it if the attempt fails then the connection will eventually close from python garbage collection anyway (trying to close it explicitly just seems like a good practice). 

I don't know logging practices in flask, but that's definitely something worth figuring out.

Reply:
In fact, the Database object @erikjasso wrote has a close_connection() function that it looks safe to just call without extra condition-checking.

Reply:
Thanks for looking into that